keps/sig-cluster-lifecycle/0003-cluster-api.md

```diff
@@ -77,7 +77,7 @@ When another component needs to create or destroy virtual machines, like the nod
 
 * Should a single Kubernetes cluster only house definitions for itself?
    * If so, that removes the ability to have a single cluster control the reconciliation of infrastructure for other clusters.
-   * However, with the concurrent [Cluster Registry](https://docs.google.com/a/google.com/document/d/1Oi9EO3Jwtp69obakl-9YpLkP764GZzsz95XJlX1a960/edit) project, a good separation of responsibilities would be that the Cluster Registry API is responsible for indexing multiple clusters, each of which would only have to know about itself. In order to achieve cross-cluster reconciliation, a controller would need to integrate with a Cluster Registry for discovery.
+   * However, with the concurrent [Cluster Registry](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/multicluster/cluster-registry/api-design.md) project, a good separation of responsibilities would be that the Cluster Registry API is responsible for indexing multiple clusters, each of which would only have to know about itself. In order to achieve cross-cluster reconciliation, a controller would need to integrate with a Cluster Registry for discovery.
```

Comment:
This can actually be just `(/contributors/design-proposals/multicluster/cluster-registry/api-design.md)`

Reply:
We would then have to change all the links right?